packages/devextreme/js/__internal/grids/data_grid/export/m_export.ts

```diff
@@ -666,6 +673,13 @@ export class ExportController extends dataGridCore.ViewController {
     }
   }
 
+  optionChanged(args) {
+    super.optionChanged(args);
+    if (args.fullName === 'export.enabled') {
```

Comment:
If I change the option like this:
`dataGridInstance.option('export', { enabled: true })`

Will the error be thrown?